src/Symfony/Component/Validator/Tests/Constraints/AbstractComparisonTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use PHPUnit\Framework\Attributes\IgnoreDeprecations;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Validator\Constraints\AbstractComparison;
+
+class AbstractComparisonTest extends TestCase
+{
+    #[IgnoreDeprecations]
```

Comment:
We also need to add it to the legacy group:


```suggestion
    #[IgnoreDeprecations]
    #[Group('legacy')]
```